include/qVoxFallTools.h

```diff
@@ -50,13 +52,13 @@ class qVoxFallTransform
 		const Vector3Tpl<float> Y(-std::sin(zRot), std::cos(zRot), 0);
 		const Vector3Tpl<float> Z(0, 0, 1);
 		const Vector3Tpl<float> Tr(0, 0, 0);
-		matrix = &ccGLMatrix::ccGLMatrix(X, Y, Z, Tr);
+		matrix = new ccGLMatrix(X, Y, Z, Tr);

```

Comment:
don't we need a destructor to free these matrices now?

Reply:
Or maybe convert these ccGLMatrix* pointers to plain instances (the impact on the code might be bigger though).

Reply:
Thanks. I Thought it was already done into the object destructor.
Yes, plain instance should make more sense, I have to track usage of these matrices in the other parts of the code.